plugin/evm/vm.go

```diff
@@ -177,6 +177,8 @@ var legacyApiNames = map[string]string{
 // VM implements the snowman.ChainVM interface
 type VM struct {
 	ctx *snow.Context
+	// snow.Context.Lock is deprecated and should not be used in rpcchainvm.
+	chainLock sync.RWMutex
```

Comment:
should we call this `contextLock`? and maybe update the comment like:
```suggestion
	// contextLock is used to coordinate global VM operations. This can be used safely instead of snow.Context.Lock which is deprecated and should not be used in rpcchainvm.
	contextLock sync.RWMutex
```

Reply:
context might be confused with ctx, but we can call it vmLock.